### ISSUE TITLE
Update org agenda files missing today with org-extend-today-until

### ISSUE THREAD

**Issue by zachgk** [bastibe/org-journal#229]
**Describe the bug**
I use the agenda integration and also have org-extend-today-until which makes the day only reset at at some point past midnight. Journal does integrate with the org-extend-today-until so it will continue to use the journal until the designated time. However, after midnight, the current days journal (technically yesterdays) is no longer added to the list of agenda files. So, I will not see those items when looking at the total agenda list.

**To Reproduce**
Use agenda integration and org-extend-today-until. The current journal should have some todo items. Then, run the agenda list sometime between midnight and your extend-today-until time and you should not see any of the todos from the current journal.

**Expected behavior**
You should see the todos from the current journal even past midnight.

**Desktop (please complete the following information):**
 - OS: arch linux
 - Emacs Version 26.3

**Your Emacs Configuration**
I use spacemacs with the following relevant variables set:
```
          org-enable-org-journal-support t
          org-journal-date-prefix "#+TITLE: "
          org-journal-file-format "%Y-%m-%d.org"
          org-journal-date-format "%A, %d %B %Y"
          org-extend-today-until 5
          org-journal-enable-agenda-integration t
```

**Comment by cslux** [comment]
Hey, thank you for the report. That shouldn't work indeed. org-journal-update-org-agenda-files does not honer org-extend-today-until.

**Comment by cslux** [comment]
@zachgk Should be fixed via 0bcefa5. Hopefully 😬
